fitbenchmarking/fitbenchmarking
# Issue #287: Replace example script with bin/fitbenchmarking and add simple argparser

#### Description of Work

Fix #287 

Moves example script to a fitbenchmarking script in a bin directory.
Updates the setup.py to install this script.
Moves code to a lib directory.
Implements very basic argparser.

#### Testing Instructions

1. Try with custom options and check results match expectations
2. Ensure selection of problem files behaves in a sensible way

Function: Does the change do what it's supposed to?
Tests: Does it pass? Is there adequate coverage for new code?
Style: Is the coding style consistent? Is anything overly confusing?
Documentation: Is there a suitable change to documentation for this change?
